src/bin/vip-import-sql.js

```diff
@@ -38,79 +41,93 @@ const err = message => {
 	process.exit( 1 );
 };
 
+const debug = debugLib( 'vip:vip-import-sql' );
+
 command( {
 	appContext: true,
 	appQuery,
 	requiredArgs: 1, // TODO print proper usage example
 	envContext: true,
 	module: 'import-sql',
 	requireConfirm: 'Are you sure you want to import the contents of the provided SQL file?',
-} ).argv( process.argv, async ( arg, opts ) => {
-	const { app, env } = opts;
-	const [ fileName ] = arg;
-
-	const trackEventWithEnv = async ( eventName, eventProps = {} ) =>
-		trackEvent( eventName, { ...eventProps, appId: env.appId, envId: env.id } );
+} ).option( 'search-replace', 'Specify the <from> and <to> pairs to be replaced' )
+	.option( 'in-place', 'Perform the search and replace explicitly on the input file' )
+	.argv( process.argv, async ( arg, opts ) => {
+		const { app, env, searchReplace } = opts;
+		const primaryDomainName = env.primaryDomain.name;
+		const [ fileName ] = arg;
 
-	await trackEventWithEnv( 'import_sql_command_execute' );
+		const trackEventWithEnv = async ( eventName, eventProps = {} ) =>
+			trackEvent( eventName, { ...eventProps, appId: env.appId, envId: env.id } );
 
-	console.log( '** Welcome to the WPVIP Site SQL Importer! **\n' );
+		await trackEventWithEnv( 'import_sql_command_execute' );
 
-	if ( ! currentUserCanImportForApp( app ) ) {
-		err( 'The currently authenticated account does not have permission to perform a SQL import.' );
-	}
+		console.log( '** Welcome to the WPVIP Site SQL Importer! **\n' );
 
-	if ( ! isSupportedApp( app ) ) {
-		await trackEventWithEnv( 'import_sql_command_error', { errorType: 'unsupported-app' } );
-		err( 'The type of application you specified does not currently support SQL imports.' );
-	}
+		debug( 'Options: ', opts );
+		debug( 'Args: ', arg );
 
-	await validate( fileName, true );
+		if ( ! isSupportedApp( app ) ) {
+			await trackEventWithEnv( 'import_sql_command_error', { errorType: 'unsupported-app' } );
+			err( 'The type of application you specified does not currently support SQL imports.' );
+		}
 
-	/**
-	 * TODO: We should check for various site locks (including importing) prior to the upload.
-	 */
+		const inputFile = await searchAndReplace( fileName, searchReplace, { isImport: true, inPlace: opts.inPlace } );
```

Comment:
The option input is described as `in-place` above...does it translate to camelCase automatically?

Additionally, I just want to make sure it defaults to `false` and the original input file isn't clobbered inadvertently.

Reply:
That's correct.  `--in-place` becomes `inPlace`, and similarly `--search-replace` becomes `searchReplace`.  I couldn't find this in the args README (docs) but I logged it out to check.  This occurs in commander as well.

I checked while testing and omitting `--in-place` results in line https://github.com/Automattic/vip/pull/605/files#diff-b92d26906cc64aca57a3c65329e117834c0c66339d30de1959ed88e6efadd2feR83 using the `outputFileReplacement` logic.  Including it results in the `inPlaceReplacement` logic being executed.